### PR TITLE
HHH-14794 : More changes to support SchemaMigrator/SchemaValidator using Hibernate Reactive

### PR DESCRIPTION
**Pull request by gbadner**
https://hibernate.atlassian.net/browse/HHH-14794

### CODE REVIEW

**Comment by Sanne** [comment]
Hi @gbadner ! Looks fine - is it intended to stay as a draft? 

We can release more frequently if that helps with the work on the HR side, let me know.

**Comment by gavinking** [comment]
@Sanne I personally would love to see that.

File: hibernate-core/src/main/java/org/hibernate/dialect/SQLServerDialect.java [hunk 1]
```diff
@@ -248,4 +251,10 @@ public String getCreateTemporaryTableColumnAnnotation(int sqlTypeCode) {
 				return "";
 		}
 	}
+
+	@Override
+	public NameQualifierSupport getNameQualifierSupport() {
```
Comment (Sanne) [inline hibernate-core/src/main/java/org/hibernate/dialect/SQLServerDialect.java:256]: Interesting that this was missing. Perhaps it deserves its own JIRA so to highlight the change better in changelogs?

**Comment by Sanne** [comment]
@gbadner I haven't heard back, but this looks totally fine so I'll start merging this. If you have more changes please send more PRs :)

**Pull request merged**